lib/EarlyFunctionAnalysis/DetectABI.cpp

```diff
@@ -488,6 +507,81 @@ void DetectABI::interproceduralPropagation() {
   }
 }
 
+void DetectABI::propagatePrototypesInFunction(model::Function &Function) {
+  const MetaAddress &Entry = Function.Entry();
+
+  FunctionSummary &Summary = Oracle.getLocalFunction(Entry);
+  ABIAnalyses::ABIAnalysesResults &ABI = Summary.ABIResults;
+  SortedVector<efa::BasicBlock> &CFG = Summary.CFG;
+  std::set<llvm::GlobalVariable *> &WrittenRegisters = Summary.WrittenRegisters;
+
+  const bool HasArguments = ABI.ArgumentsRegisters.size() > 0;
+  const bool HasReturns = ABI.FinalReturnValuesRegisters.size() > 0;
+  const bool IsSingleNode = CFG.size() == 1;
```

Comment:
Drop the `const`, one day `const` will be default, but for now the return on putting `const` like this is too little compared to the benefit.